kong/plugins/rate-limiting/policies/init.lua

```diff
@@ -51,6 +51,15 @@ local get_local_key = function(conf, identifier, period_date, name)
 end
 
 
+local function get_redis_pool_name(conf)
+  -- return a special pool name only if redis_database is set to non-zero
+  -- otherwise use the default pool name host:port
+  if conf.redis_database ~= 0 then 
+    local key = fmt("%s:%d:%d", conf.redis_port, conf.redis_port, conf.redis_database)
```

Comment:
The second argument here should be `conf.redis_host`

Reply:
Additionally, prior art has proven that a simple concat operation could perform as well or better than `string.format`, we should maybe follow this new rule of thumb.